### ISSUE TITLE
(marja): activate once parsing 'return' is merged into ParserBase. 

### ISSUE THREAD

**Issue by piascikj** [imdone/v8#1889]
Issue opened with imdone.io from code comment on [49c6287](https://github.com/imdone/v8/commit/49c62872dbd11d16373c3e894257348c33267bab)

TODO (marja): activate once parsing 'return' is merged into ParserBase. id:1881

----

https://github.com/imdone/v8/blob/49c62872dbd11d16373c3e894257348c33267bab/test/cctest/test-parsing.cc#L1519-L1524
